### ISSUE TITLE
Incompatible with uws (and some browsers?)

### ISSUE THREAD

**Issue by tomsun** [baygeldin/ws-streamify#1]
Just FYI, getting:
```
Error in socket handler TypeError: socket.addEventListener is not a function
    at new WebSocketStream (/opt/node_modules/ws-streamify/lib/index.js:71:12)
```

...when using https://www.npmjs.com/package/uws - not an issue if switching to https://www.npmjs.com/package/ws

Observed with ws-streamify@0.1.1 and uws@0.13.0

Possibly semi-related discussion:
https://github.com/maxogden/websocket-stream/issues/87
https://github.com/maxogden/websocket-stream/pull/88

**Comment by baygeldin** [comment]
Hi! Thanks for the info. I changed the methods to the ones that are in MDN documentation and released 0.1.2. Hope it helps.

**Comment by tomsun** [comment]
@baygeldin awesome, thanks!
